compiler/tools/suggest.nim

```diff
@@ -49,6 +49,10 @@ import
     types,
     typesrenderer,
     wordrecg,
+    syntaxes,
```

Comment:
For reading convenience, please keep the imports in ascending alphabetical order.